modules/ROOT/pages/release-notes.adoc

```diff
@@ -4,4 +4,25 @@
 [NOTE]
 ====
 The 2023.1 is in development.
-====
\ No newline at end of file
+====
+
+== Feature deprecations and removals
+
+=== Removals
+
+==== Multi-tenancy removal
+
+Deprecated in Bonita 2021.1, xref:2022.2@ROOT:multi-tenancy-and-tenant-configuration.adoc[multi-tenancy] feature has been technically removed.
```

Comment:
```suggestion
Deprecated since Bonita 2021.1, xref:2022.2@ROOT:multi-tenancy-and-tenant-configuration.adoc[multi-tenancy] feature has been technically removed.
```